__docs__/wonder-blocks-form/text-field.stories.tsx

```diff
@@ -702,6 +703,68 @@ ReadOnly.parameters = {
     },
 };
 
+export const WithAutofocus: StoryComponentType = () => {
+    const [value, setValue] = React.useState("");
+    const [showDemo, setShowDemo] = React.useState(false);
+
+    const handleChange = (newValue: string) => {
+        setValue(newValue);
+    };
+
+    const handleKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
+        if (event.key === "Enter") {
+            event.currentTarget.blur();
+        }
+    };
+
+    const handleShowDemo = () => {
+        setShowDemo(!showDemo);
+    };
+
+    const AutoFocusDemo = () => (
+        <View style={{flexDirection: "row"}}>
+            <Button onClick={() => {}}>Some other focusable element</Button>
+            <TextField
+                id="tf-13"
+                value={value}
+                placeholder="Placeholder"
+                autoFocus={true}
+                onChange={handleChange}
+                onKeyDown={handleKeyDown}
+                style={{flexGrow: 1, marginLeft: 12}}
```

Comment:
**suggestion:** I'd still recommend using WB components as much as we can, so other devs could see how we use it from the examples in the site.

```suggestion
                style={{flexGrow: 1, marginLeft: Spacing.small_12}}
```